nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-processors/src/test/java/org/apache/nifi/processors/iceberg/TestIcebergRecordConverter.java

```diff
@@ -433,11 +494,11 @@ private static Record setupChoiceTestRecord() {
     }
 
     @DisabledOnOs(WINDOWS)
-    @ParameterizedTest
```

Comment:
Please don't remove `@ParameterizedTest` in this test class since it is testing different writers for different format types.